packages/manager/apps/dedicated/client/app/dedicated/server/install/image/index.js

```diff
@@ -0,0 +1,34 @@
+import angular from 'angular';
+
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+import ngUiRouterBreadcrumb from '@ovh-ux/ng-ui-router-breadcrumb';
```

Comment:
❤️

Reply:
:-) This is just imported for this part. Should be moved once all the dedicated section will use global breadcrumb